samples/RT-sample-app/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,130 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json",
+    "manifestVersion": "devPreview",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "https://www.example.com",
+        "privacyUrl": "https://www.example.com/privacy",
+        "termsOfUseUrl": "https://www.example.com/termofuse"
+    },
+    "icons": {
+        "color": "color.png",
+        "outline": "outline.png"
+    },
+    "name": {
+        "short": "RT-Sample-App",
+        "full": "full name for RT-sample-app"
+    },
+    "description": {
+        "short": "Specify runtime requirements for specific app capabilities",
+        "full": "Specify runtime requirements for specific app capabilities"
+    },
+    "accentColor": "#FFFFFF",
+    "elementRelationshipSet": {
+        "oneWayDependencies": [
```

Comment:
We'll need one more - static tab dependsOn composeExtension. This is a very common scenario various existing Teams app have that's blocking them from going multi-host, including our own C4S 1P app.

Reply:
Is the way I added this correct? Inside the one-way dependency, there's another element where the Static Tab depends on the Compose-Extension.
![image](https://github.com/user-attachments/assets/c850e946-f17a-4f26-b16f-06b397ab536e)

I tried the approach mentioned above, but it resulted in a manifest parsing error because the Compose Extension ID was used twice within the one-way dependency.

After using another elementRelationshipSet, it worked for me as shown below.
<img width="377" alt="image" src="https://github.com/user-attachments/assets/984c583e-5c4d-4e38-bf03-ec0fcb7d7946">

Provide your feedbacks!

**Thanks!!**

Reply:
**Parsing Error :** 
Dependency from element staticTab_Id to element **composeExt_ID** exists in both one-way and mutual dependencies. Dependency from element **composeExt_ID** to element exists in both one-way and mutual dependencies.

Reply:
That's actually correct error message. You can't have a relationship defined in both oneWay and mutual. So e.g. if you already specify your tab 1 oneWayDependsOn CE 1, then in mutual you shouldn't also include tab 1 and CE1, as that'll fail validation rule.

Reply:
I guess, In this case we should maintain multiple appManifest to cover all hero's scenarios.